contrib/inventory/gce.py

```diff
@@ -283,6 +295,11 @@ def group_instances(self):
         meta["hostvars"] = {}
 
         for node in self.driver.list_nodes():
+
+            # check desired states from config file
```

Comment:
Maybe a more detailed comment about what it's doing here?  The ultimate behavior is what the user expects, but the code to implement might not be.  Worth explaining that we only check for the state in the list IF the list is actually populated.